pandapower/opf/PpPmInterface/src/pm_models/run_powermodels_mn_storage.jl

```diff
@@ -36,9 +35,9 @@ function set_pq_values_from_timeseries(mn, time_series)
     return mn
 end
 
-function run_powermodels(json_path)
+function run_powermodels_mn_storage(json_path)
```

Comment:
only focus on run_powermodels.jl file. after approving the first version of the package, we discuss other models.